jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/deadlines/notifications/impl/email/EmailNotificationListener.java

```diff
@@ -60,19 +60,14 @@
     
     private static final Logger logger = LoggerFactory.getLogger(EmailNotificationListener.class);
 
-    private UserInfo userInfo;
-
     private Session mailSession = EmailSessionProducer.produceSession();
     
-    public EmailNotificationListener(UserInfo userInfo) {
-    	this.userInfo = userInfo;
-    }
     
     @Override
-    public void onNotification(NotificationEvent event) {
-
+    public void onNotification(NotificationEvent event, Object... params) {
+        UserInfo userInfo = (UserInfo) params[0];
```

Comment:
thanks @mrietveld, the more I think about it the more I am convinced this was not the best idea :) I'll stick to UserInfo only as this is rather important from notification point of view and more parameters can be given as variables of NotificationEvent class. How does that sound?